app/assets/stylesheets/public/_docs.scss

```
@@ -419,6 +421,9 @@ label[for='navicon'] span { // hide alt text
     top: 0;
 
     .Docs__toc__sticky {
+      background: #fefefe;
+      box-shadow: 2px 2px 4px 5px #eee;
```

```suggestion
      box-shadow: 2px 2px 5px 3px rgba(0 0 0 / 10%);
```

2 tweaks please 💄 💅
- Soften the box-shadow with more blur than spread (5px then 4px)
- Use transparency in shadow over opaque colour (replace `#eee` with `rgba(0 0 0 / 10%)`)

---

Looks like there is an issue with that one @olyism 

```
May 20 08:32:09 AM   #18 4.204 SassC::SyntaxError: Error: overloaded function `rgba` given wrong number of arguments
May 20 08:32:09 AM   #18 4.204         on line 425:35 of app/assets/stylesheets/public/_docs.scss
May 20 08:32:09 AM   #18 4.204         from line 4:9 of app/assets/stylesheets/application.scss
May 20 08:32:09 AM   #18 4.204 >>       box-shadow: 2px 2px 5px 3px rgba(0 0 0 / 10%);
May 20 08:32:09 AM   #18 4.204
May 20 08:32:09 AM   #18 4.204    ----------------------------------^
May 20 08:32:09 AM   #18 4.204 /app/app/assets/stylesheets/public/_docs.scss:425
May 20 08:32:09 AM   #18 4.204 /usr/local/bundle/gems/sassc-2.4.0/lib/sassc/engine.rb:50:in `render'
```